### PR TITLE
Add an example for creating an automation to mark zombie flow runs as crashed

### PR DESCRIPTION
**Pull request by desertaxle**
This PR adds an example of how to add an automation to mark zombie flow runs as crashed. I had trouble finding a good place to put this, so I'm open to putting this somewhere else.

### CODE REVIEW

File: docs/v3/automate/events/automations-triggers.mdx [hunk 1]
```diff
@@ -905,6 +905,60 @@ In the above example:
 - `user_id_1` creates and then completes an order, triggering a run of our deployment.
 - `user_id_2` creates an order, but no completed event is emitted so no deployment is triggered.
 
+### Detect and respond to zombie flows
+
+If the infrastructure a flow is running on suddenly fails (e.g. the machine crashes or a container is evicted), 
+Prefect's orchestration engine will be unable to report state changes and the flow run will get stuck in the running state.
+
+Fortunately, flow runs triggered via deployment can emit heartbeats as they are running, and a Prefect automation can update 
+a flow run's state to crashed if the server stops receiving heartbeats for a flow run.
```
Comment (cicdw) [inline docs/v3/automate/events/automations-triggers.mdx:914]:
```suggestion
a flow run's state to crashed if the server stops receiving heartbeats for that flow run.
```

File: docs/v3/automate/events/automations-triggers.mdx [hunk 2]
```diff
@@ -905,6 +905,60 @@ In the above example:
...
+
+<Note>
+**Enable flow run heartbeat events**
+
+You will need to ensure you're running Prefect version 3.1.8 or greater and set `PREFECT_RUNNER_HEARTBEAT_FREQUENCY` to emit flow run heartbeat events.
```
Comment (cicdw) [inline docs/v3/automate/events/automations-triggers.mdx:919]: worth including "set to an integer / to an integer greater than 30"?

**Review by cicdw — COMMENTED** [review]
minor comments, this is great

**Review by discdiver — APPROVED** [review]

File: docs/v3/automate/events/automations-triggers.mdx [hunk 3]
```diff
@@ -905,6 +905,68 @@ In the above example:
 - `user_id_1` creates and then completes an order, triggering a run of our deployment.
 - `user_id_2` creates an order, but no completed event is emitted so no deployment is triggered.
 
+### Detect and respond to zombie flows
+
+If the infrastructure a flow is running on suddenly fails (e.g. the machine crashes or a container is evicted),
```
Comment (discdiver) [inline docs/v3/automate/events/automations-triggers.mdx:910]:
```suggestion
If the infrastructure a flow is running on suddenly fails (for example, if the machine crashes or a container is evicted), 
```

File: docs/v3/automate/events/automations-triggers.mdx [hunk 4]
```diff
@@ -905,6 +905,68 @@ In the above example:
...
+To create an automation that marks zombie flow runs as crashed, run this script:
+```python
+from datetime import timedelta
+
+from prefect.automations import Automation
+from prefect.client.schemas.objects import StateType
+from prefect.events.actions import ChangeFlowRunState
+from prefect.events.schemas.automations import EventTrigger, Posture
+from prefect.events.schemas.events import ResourceSpecification
+
```
Comment (discdiver) [inline docs/v3/automate/events/automations-triggers.mdx:932]:
```suggestion


```

File: docs/v3/automate/events/automations-triggers.mdx [hunk 5]
```diff
@@ -905,6 +905,68 @@ In the above example:
...
+    ),
+    actions=[
+        ChangeFlowRunState(
+            state=StateType.CRASHED,
+            message="Flow run marked as crashed due to missing heartbeats.",
+        )
+    ],
+)
+
+my_automation.create()
```
Comment (discdiver) [inline docs/v3/automate/events/automations-triggers.mdx:958]:
```suggestion
if __name__=="__main__":
    my_automation.create()
```

**Review by discdiver — APPROVED** [review]
Thank you! Added a few suggestions for style consitency.

**Pull request merged**